src/assets/styles/base.scss

```diff
@@ -76,7 +76,6 @@ html {
 .app-blue {
   color: $app-blue !important;
 }
-
```

Comment:
Most other classes in here are separated by a blank line, so please revert this deletion.